pkg/controller/cloud/node_controller.go

```diff
@@ -148,6 +148,14 @@ func (cnc *CloudNodeController) updateNodeAddress(node *v1.Node, instances cloud
 		return
 	}
 
+	//Node not present according the cloud provider, no need to update node address.
+	if _, err := instances.ExternalID(types.NodeName(node.Name)); err != nil {
+		if err == cloudprovider.InstanceNotFound {
+			glog.V(2).Infof("this node %s no longer present in cloud provider, and no process.", node.Name)
```

Comment:
nit: `the node %s is no longer present according to the cloud provider, do not process`

Reply:
Thanks your review,  I will fix it.